### DISCUSSION TITLE
DZRJ-DTV

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#651] [Q&A]
**Channel Name:** RJ DigiTV
**Channel ID:** RJDigiTV.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Free to air channel in the Philippines.

**Comment by BellezaEmporium** [comment]
https://stream.rjtv.stream/live/smil:rjtv.smil/playlist.m3u8 ?

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19923
